examples/linesplitter/CMakeLists.txt

```diff
@@ -15,8 +15,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
+cmake_minimum_required(VERSION 3.24)
```

Comment:
Sure feel free to flip it back or lower it 'per policy'.  For me as a CMake neophyte it was the cheapest way to silence this:

```-- Building using CMake version: 3.27.4
CMake Warning (dev) at /usr/share/cmake-3.27/Modules/FetchContent.cmake:1316 (message):
  The DOWNLOAD_EXTRACT_TIMESTAMP option was not given and policy CMP0135 is
  not set.  The policy's OLD behavior will be used.  When using a URL
  download, the timestamps of extracted files should preferably be that of         
  the time of extraction, otherwise code that depends on the extracted
  contents might not be rebuilt if the URL changes.  The OLD behavior
  preserves the timestamps from the archive instead, but this is usually not
  what you want.  Update your project to the NEW behavior or specify the
  DOWNLOAD_EXTRACT_TIMESTAMP option with a value of true to avoid this
  robustness issue.                                 
Call Stack (most recent call first):
  CMakeLists.txt:38 (fetchcontent_declare)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found Python: /usr/bin/python3 (found version "3.11.6") found components: Interpreter 
```

which trust you know how to silence otherwise.  By all means commit into the branch (if the remainder is of use to you) to address this otherwise.

Reply:
That was easy enough so I just attempted a cleanup.  Back to 3.14 and now a policy setter.  Should have looked on directory up...